fluentd/fluent.conf

```diff
@@ -135,18 +134,36 @@
     bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
     ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
   </filter>
+  <filter **kibana**>
```

Comment:
We certainly want a consistent key; I just don't understand why kibana gets
specific treatment. Why doesn't this apply for every container's logs? They
are all coming out of Docker in the same format.

On Tue, Sep 29, 2015 at 1:59 PM, Eric Wolinetz <notifications@github.com>
wrote:

> In fluentd/fluent.conf
> <https://github.com/openshift/origin-aggregated-logging/pull/4#discussion_r40705480>
> :
>
> > @@ -135,18 +134,36 @@
> >      bearer_token_file /var/run/secrets/kubernetes.io/serviceaccount/token
> >      ca_file /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
> >    </filter>
> > +  <filter **kibana**>
>
> This is also part of the ECL initiative. The filter in this case matches
> on the tag, which for this path is the [container] file name. The thing we
> gain from this is that any log messages will all end up with the same key
> otherwise we would have 'message', 'msg', 'log', and 'err' that all
> contained messages we would want to correlate with.
>
> I'm not sure if there would be any more restrictive of a match that we can
> put here since anyone could name their container *kibana* and be matched
> here, I'll verify whether or not anything negative happens to application
> logs that aren't coming from kibana are filtered through here (e.g. loss of
> data).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/4/files#r40705480>
> .
>